src/data/full-quiz.js

```diff
@@ -5,71 +5,71 @@ const fullQuiz = [
     Distractor1: "undefined",
     Distractor2: "null",
     Distractor3: "Syntax error",
-    Explanation: "The nullish coalesce operator in Javascript returns first expression if it is not null or undefined, else it returns second expression.",
+    Explanation: "The nullish coalescing operator in JavaScript will return the right-hand side operand if the left-hand side operand is null or undefined.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator",
   },
   {
-    Question: "What object does this keyword point to in global execution context?",
+    Question: "In the global execution context, 'this' refers to the ______ object whether in strict mode or not.",
     Answer: "window",
     Distractor1: "null",
     Distractor2: "undefined",
     Distractor3: "document",
-    Explanation: "In global execution context, this keyword always points to window object",
+    Explanation: "In the global execution context, the 'this' keyword always points to the window object",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this",
   },
   {
-    Question: "Which queue is emptied first by Javascript event loop?",
+    Question: "Which queue is emptied first by an event loop in JavaScript?",
     Answer: "Microtask queue",
     Distractor1: "Callback queue (Macrotask queue)",
     Distractor2: "None",
     Distractor3: "Both",
-    Explanation: "Event loop first empties the Microtask queue and once its empty, starts to empty Callback queue",
+    Explanation: "An event loop first empties the Microtask queue and once it is empty it starts to empty the Callback queue.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/EventLoop",
   },
   {
-    Question: "Which of the following is not in Temporal Dead Zone(TDZ) during hoisting?",
+    Question: "Which of the following is NOT in the Temporal Dead Zone(TDZ) during hoisting?",
     Answer: "var",
     Distractor1: "let",
     Distractor2: "const",
     Distractor3: "class",
-    Explanation: "var is hoisted with default value of undefined. While let, const and classes are hoisted but are in Temporal Dead Zone(TDZ) until declaration is executed.",
+    Explanation: "var is hoisted with the default value of undefined while let, const and classes are hoisted but are in the Temporal Dead Zone(TDZ) until the declaration is executed.",
     Link: "https://developer.mozilla.org/en-US/docs/Glossary/Hoisting",
   },
   {
-    Question: "Which of the following expression will return true in Javascript?",
+    Question: "Which of the following expression will return true in JavaScript?",

```

Comment:
expression should be expressions.
Which of the following expressions will return true in JavaScript?

Reply:
Done!